pyrit/models/prompt_request_response.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pyrit.models import PromptRequestPiece
+
+
+class PromptRequestResponse:
+    def __init__(self, request_pieces: list[PromptRequestPiece]):
+        self.request_pieces = request_pieces
+
+    def __str__(self):
+        ret = ""
+        for request_piece in self.request_pieces:
+            ret += str(request_piece) + "\n"
+        return ret
```

Comment:
```suggestion
        ret = ""
        for request_piece in self.request_pieces:
            ret += str(request_piece) + "\n"
        return "\n".join([str(request_piece) for request_piece in self.request_pieces])
```
Just a suggestion... IMO at least as readable as the loop here, and more pythonic. Not a hill I will die on, though 😄